### PR TITLE
Add wrapper for utils package from sriov-device-plugin

### PR DESCRIPTION
**Pull request by ykulazhenkov**
This is a pre requisite to cover code in host pkg with unit tests.

cc @adrianchiris @SchSeba @zeeke

### CODE REVIEW

File: pkg/host/internal/lib/dputils/dputils.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import (
+	dputils "github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/utils"
+)
+
+func New() DPUtilsLib {
+	return &libWrapper{}
+}
+
+//go:generate ../../../../../bin/mockgen -destination mock/mock_dputils.go -source dputils.go
+type DPUtilsLib interface {
```
Comment (adrianchiris) [inline pkg/host/internal/lib/dputils/dputils.go:12]: These might be good candidates for sriovnet (not related to this PR)

**Review by adrianchiris — APPROVED** [review]
LGTM

**Comment by adrianchiris** [comment]
@zeeke can we merge this one ?

**Review by zeeke — APPROVED** [review]
LGTM

**Pull request merged**